src/settings/networkremotesettingspage.cpp

```diff
@@ -0,0 +1,99 @@
+#include <QStyle>
+#include "core/iconloader.h"
+#include "core/logging.h"
+#include "networkremote/networkremote.h"
+#include "settings/settingsdialog.h"
+#include "settings/networkremotesettingspage.h"
+#include "ui_networkremotesettingspage.h"
+
+using namespace Qt::Literals::StringLiterals;
+
+NetworkRemoteSettingsPage::NetworkRemoteSettingsPage(SettingsDialog *dialog, QWidget *parent) :
+  SettingsPage(dialog,parent),
+  ui_(new Ui_NetworkRemoteSettingsPage),
+  settings_(new NetworkRemoteSettings)
+{
+  ui_->setupUi(this);
+  const int iconSize = style()->pixelMetric(QStyle::PM_TabBarIconSize);
+  setWindowIcon(IconLoader::Load(QStringLiteral("network-remote"), true, 0,iconSize));
```

Comment:
```suggestion
  setWindowIcon(IconLoader::Load(u"network-remote"_s), true, 0, style()->pixelMetric(QStyle::PM_TabBarIconSize)));
```

Reply:
Gives the following warning in Qt Creator
networkremotesettingspage.cpp:39:56: Too many arguments to function call, expected single argument 'icon', have 4 arguments
qwidget.h:378:10: 'setWindowIcon' declared here